<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainView</class>
 <widget class="QDialog" name="mainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>59</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stu_id">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="exitSystem">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>510</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearTable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Table</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>541</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Item</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previewTable">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitDb">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>510</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnItems">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>RETURN ITEMS</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>201</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="itemsList">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Select One</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rugby Balls</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hockey Balls</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Table Tennis Bats</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tennis Balls</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>59</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="getItemCount">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="editTeams">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>EDIT TEAMS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editItems">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>EDIT ITEMS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Commands</string>
   </property>
  </widget>
  <widget class="QPushButton" name="availableIventory">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>AVAILABLE INVENTORY</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>                   SPORTS INVENTORY MAIN VIEW</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stu_name">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editTeams_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>RETURN ITEMS</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
